src/okta/__recordings__/delayNextApiRequest_2022936668/recording.har

```diff
@@ -0,0 +1,366 @@
"Where did you meet your spouse/significant other?\"},\"provider\":{\"type\":\"OKTA\",\"name\":\"OKTA\"}},\"_links\":{\"self\":{\"href\":\"https://dev-857255.okta.com/api/v1/users/00us5to77GW1mVs6w4x6\"}}},{\"id\":\"00us5ty22kvu4NAxo4x6\",\"status\":\"ACTIVE\",\"created\":\"2020-08-27T01:10:40.000Z\",\"activated\":\"2020-08-27T01:10:47.000Z\",\"statusChanged\":\"2020-08-27T11:32:46.000Z\",\"lastLogin\":\"2020-11-09T22:21:23.000Z\",\"lastUpdated\":\"2020-11-09T22:21:23.000Z\",\"passwordChanged\":\"2020-11-09T22:21:23.000Z\",\"type\":{\"id\":\"otys5to4jqDf0Ws0W4x6\"},\"profile\":{\"firstName\":\"Austin\",\"lastName\":\"Kelleher\",\"mobilePhone\":null,\"secondEmail\":null,\"login\":\"austin.kelleher@jupiterone.com\",\"email\":\"austin.kelleher@jupiterone.com\"},\"credentials\":{\"password\":{},\"emails\":[{\"value\":\"austin.kelleher@jupiterone.com\",\"status\":\"VERIFIED\",\"type\":\"PRIMARY\"}],\"recovery_question\":{\"question\":\"What was your dream job as a child?\"},\"provider\":{\"type\":\"OKTA\",\"name\":\"OKTA\"}},\"_links\":{\"self\":{\"href\":\"https://dev-857255.okta.com/api/v1/users/00us5ty22kvu4NAxo4x6\"}}},{\"id\":\"00us5z1aerBUBP41M4x6\",\"status\":\"ACTIVE\",\"created\":\"2020-08-27T01:10:14.000Z\",\"activated\":\"2020-08-27T01:10:15.000Z\",\"statusChanged\":\"2020-08-27T14:59:28.000Z\",\"lastLogin\":\"2020-08-27T14:59:28.000Z\",\"lastUpdated\":\"2020-08-27T14:59:28.000Z\",\"passwordChanged\":\"2020-08-27T14:59:28.000Z\",\"type\":{\"id\":\"otys5to4jqDf0Ws0W4x6\"},\"profile\":{\"firstName\":\"Akash\",\"lastName\":\"Ganapathi\",\"mobilePhone\":null,\"secondEmail\":null,\"login\":\"akash.ganapathi@jupiterone.com\",\"email\":\"akash.ganapathi@jupiterone.com\"},\"credentials\":{\"password\":{},\"emails\":[{\"value\":\"akash.ganapathi@jupiterone.com\",\"status\":\"VERIFIED\",\"type\":\"PRIMARY\"}],\"recovery_question\":{\"question\":\"What is the name of your first stuffed animal?\"},\"provider\":{\"type\":\"OKTA\",\"name\":\"OKTA\"}},\"_links\":{\"self\":{\"href\":\"https://dev-857255.okta.com/api/v1/users/00us5z1aerBUBP41M4x6\"}}},{\"id\":\"00us5zkpf7nGTeKXu4x6\",\"status\":\"ACTIVE\",\"created\":\"2020-08-27T01:09:43.000Z\",\"activated\":\"2020-08-27T01:09:43.000Z\",\"statusChanged\":\"2020-08-27T15:09:23.000Z\",\"lastLogin\":\"2020-12-02T17:06:34.000Z\",\"lastUpdated\":\"2020-08-27T15:09:23.000Z\",\"passwordChanged\":\"2020-08-27T15:09:23.000Z\",\"type\":{\"id\":\"otys5to4jqDf0Ws0W4x6\"},\"profile\":{\"firstName\":\"Adam\",\"lastName\":\"Williams\",\"mobilePhone\":null,\"secondEmail\":null,\"login\":\"adam.williams@jupiterone.com\",\"email\":\"adam.williams@jupiterone.com\"},\"credentials\":{\"password\":{},\"emails\":[{\"value\":\"adam.williams@jupiterone.com\",\"status\":\"VERIFIED\",\"type\":\"PRIMARY\"}],\"recovery_question\":{\"question\":\"What was your dream job as a child?\"},\"provider\":{\"type\":\"OKTA\",\"name\":\"OKTA\"}},\"_links\":{\"self\":{\"href\":\"https://dev-857255.okta.com/api/v1/users/00us5zkpf7nGTeKXu4x6\"}}}]"
```

Comment:
Very important that we avoid adding meaningful info to our profiles in the preview account! @kevincasey1222 could you add to your list a task to redact PII from these recordings? We can talk about how to approach it and hopefully further our patterns in the recordings code of the SDK.

Reply:
Ah, yes, of course. It didn't even occur to me that the existing recording would have such things - I was more concerned with making sure I didn't cause them, and I hadn't started messing with tests and recordings yet. Given that they are in the existing Okta integration, they'll be in the git history even after I clean them up. We might need some fancy Git-fu to make that right.

Reply:
PII removed